### PR TITLE
[NUI] Enhance button relayout and change styles 

### PR DESCRIPTION
**Pull request by rabbitfor**
Signed-off-by: Jiyun Yang <ji.yang@samsung.com>

### Description of Change ###
* Enhance button relayout logic
* Change default item alignments
* Change resource to png from svg
* Selector implements IEnumerable

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI.Components/Controls/Button.Internal.cs [hunk 1]
```diff
@@ -289,6 +289,11 @@ public override void OnRelayout(Vector2 size, RelayoutContainer container)
         {
             Debug.Assert(size != null);
 
+            if (size.Equals(this.size))
```
Comment (dongsug-song) [inline src/Tizen.NUI.Components/Controls/Button.Internal.cs:292]: could you check the warning below?
I know there is `Debug.Assert(size != null);` above, maybe checker can not catch it.
could you change it as `size?.Equals(...` or suppress the warning?

File: src/Tizen.NUI.Components/Controls/Button.Internal.cs [hunk 2]
```diff
@@ -289,6 +289,11 @@ public override void OnRelayout(Vector2 size, RelayoutContainer container)
...
+            {
+                return;
+            }
+
```
Comment (dongsug-song) [inline src/Tizen.NUI.Components/Controls/Button.Internal.cs:296]: same as above. (warning at line 308)
null checking of `controlStateChangedInfo`

File: src/Tizen.NUI.Components/Controls/Button.Internal.cs (hunk 1 above)
Comment (rabbitfor) [reply src/Tizen.NUI.Components/Controls/Button.Internal.cs:292]: Done. Thank you.

File: src/Tizen.NUI.Components/Controls/Button.Internal.cs (hunk 2 above)
Comment (rabbitfor) [reply src/Tizen.NUI.Components/Controls/Button.Internal.cs:296]: Done.

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Pull request merged**